mysql/README.md

```diff
@@ -0,0 +1,96 @@
+# TDK MySQL Demo
+
+[Sakila](https://dev.mysql.com/doc/sakila/en/) is a sample database schema that can be used in MySQL tutorials, articles, demos, etc.
+
+```shell
+git clone https://github.com/synthesized-io/pagila-tdk-demo
+cd pagila-tdk-demo/mysql
+```
+
+Prepare an [inventory file](https://docs.synthesized.io/tdk/latest/user_guide/reference/inventory) - [inventory.yaml](inventory.yaml).
```

Comment:
What does the phrase "Prepare an inventory file" mean here? Should the user do something? I suggest removing it or to make it more clear.

Reply:
Fixed in [384d5ba](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/384d5ba92ac472b127fd1688589afe1251a3425f)